src/incentives/utils/create-transaction.js

```diff
@@ -1,12 +1,42 @@
 const TransactionModel = require("@models/transaction");
+const HostModel = require("@models/Host");
 const constants = require("@config/constants");
-const { logObject, logElement, logText } = require("@utils/log");
+const { logObject, logText } = require("@utils/log");
 const isEmpty = require("is-empty");
 const httpStatus = require("http-status");
-const generateFilter = require("@utils/generate-filter");
 const log4js = require("log4js");
 const logger = log4js.getLogger("create-transaction-util");
 const axios = require("axios");
+
+/*********************************** Helper Functions ***********************************/
+function createProductItemForMobileMoneyPayout(phone_number) {
+  const airtelCodes = ["075", "070"];
```

Comment:
@BenjaminSsempala , we need to find out all the valid codes for the different telecommunication networks.